llarp/linux/service_manager.cpp

```diff
@@ -1,13 +1,10 @@
 #include <llarp/util/service_manager.hpp>
```

Comment:
I think this file should be removed now?

Reply:
yes it should be removed